webcompat/form.py

```diff
@@ -25,7 +25,11 @@
 AUTH_REPORT = 'github-auth-report'
 PROXY_REPORT = 'github-proxy-report'
 SCHEMES = ('http://', 'https://')
+<<<<<<< HEAD
```

Comment:
sorry about that. i made sure to rebase against upstream before pushing 
my last commit. but if there's still conflicts, i must have missed 
something...

On 04/01/2016 01:31 PM, Mike Taylor wrote:
> In webcompat/form.py
> <https://github.com/webcompat/webcompat.com/pull/973#discussion_r58239162>:
>
>> @@ -25,7 +25,11 @@
>>  AUTH_REPORT = 'github-auth-report'
>>  PROXY_REPORT = 'github-proxy-report'
>>  SCHEMES = ('http://', 'https://')
>> +<<<<<<< HEAD
>
> Ah, this explains why my attempt at fixing conflicts was so hard. :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/webcompat/webcompat.com/pull/973/files/c3f34bd4c5451584117307294896584389cd46e0#r58239162>
>